Source/ConversationV1/Conversation.swift

```diff
@@ -61,7 +61,11 @@ public class Conversation {
             let json = try JSON(data: data)
             let error = try json.getString(at: "error")
             let code = (try? json.getInt(at: "code")) ?? 400
-            let userInfo = [NSLocalizedFailureReasonErrorKey: error]
+            let description = try? json.getString(at: "description")
+            let userInfo = [
+                NSLocalizedFailureReasonErrorKey: error,
+                NSLocalizedRecoverySuggestionErrorKey: description
```

Comment:
Ended up going with something like this:
```
var userInfo = [
                NSLocalizedFailureReasonErrorKey: error
            ]
            if let description = try? json.getString(at: "description") {
                userInfo[NSLocalizedRecoverySuggestionErrorKey] = description
            }
```

The [commit's](https://github.com/watson-developer-cloud/swift-sdk/pull/579/commits/2bdc81f3bb4df3046bbd89c77e60c9d0017c112e) attached to the latest PR to release the SDK, so feel free to pull/modify if you think there's a better way! The build passes again with these changes as well (yay).